nncf/openvino/quantization/quantize_model.py

```diff
@@ -187,14 +181,30 @@ def native_quantize_with_accuracy_control_impl(
     if advanced_accuracy_restorer_parameters.ranking_subset_size is not None:
         ranking_subset_size = advanced_accuracy_restorer_parameters.ranking_subset_size
 
+    init_params = None
+    if advanced_accuracy_restorer_parameters.tune_hyperparams:
+        init_params = {
+            "preset": preset,
+            "target_device": target_device,
+            "subset_size": subset_size,
+            "fast_bias_correction": fast_bias_correction,
+            "model_type": model_type,
+            "ignored_scope": ignored_scope,
+            "advanced_parameters": copied_parameters,
+        }
+
     accuracy_aware_loop = QuantizationAccuracyRestorer(
```

Comment:
I suggest choosing one of the following strategies:
1. One class with post training quantization with accuracy control
```suggestion
    accuracy_aware_loop = PostTrainingQuantizationWithAccuracyControl(
```
2. Implement the following flow `quantize` or `quantize_with_tune_hyperparams` ->  `QuantizationAccuracyRestorer` in `native_quantize_with_accuracy_control_impl`.

Reply:
done